dotcom-rendering/src/types/front.ts

```diff
@@ -233,6 +233,24 @@ export type DCRSnapType = {
 	embedCss?: string;
 };
 
+type FECollectionConfigType = {
+	displayName: string;
+	metadata?: { type: FEContainerPalette }[];
```

Comment:
Is this optional - or would it return an empty list (optional lists are fiddly to deal with so it's best to try and avoid them unless there's a reason)?

Reply:
I think that it is optional. This is a collection config from the current UK front:
<img width="689" alt="image" src="https://user-images.githubusercontent.com/37048459/180387982-b1c15284-f5ba-4ced-8f82-5b5e4b8372b7.png">

My understanding is that Frontend excludes keys with `None` values during serialisation, although I think you're right that there's an argument for passing an empty array when the `Option` is a list of some sort.

Reply:
Yes, we should return `Nil` instead of `None`.